ci/testkomodo.sh

```diff
@@ -72,11 +72,12 @@ start_tests () {
     fi
     if ! which bsub 2>/dev/null && basetemp=$(mktemp -d -p ~/pytest-tmp); then
         export PATH=$PATH:/opt/pbs/bin
-        if [[ $(uname -r) == *el7* ]] ; then
-            export _ERT_TESTS_DEFAULT_QUEUE_NAME=permanent
-        else
-            export _ERT_TESTS_DEFAULT_QUEUE_NAME=permanent_8
-        fi
+        echo '#!/bin/sh' > queue_tester.sh
```

Comment:
As discussed, it'd be better to have this test in `komodo-releases` and set the "working" queue as env or parameter.